server/src/main/java/io/littlehorse/common/model/getable/core/variable/VariableValueModel.java

```diff
@@ -41,6 +43,10 @@ public class VariableValueModel extends LHSerializable<VariableValue> {
 
     private ExecutionContext context;
 
+    private boolean deserializationErrorPresent = false;
```

Comment:
Rather than having two fields with `Getter`s, I would suggest that we have a method:

```
public Optional<String> getDeserializationError() {
   ...
};
```

And also make the `deserializationErrorPresent` field nave `@Getter(AccessLevel.NONE)` on it.